rootfs/Dockerfile

```diff
@@ -17,11 +17,18 @@ RUN apt-get update \
 		libssl-dev \
 		libpcre3-dev \
 		make \
-	&& export NGINX_VERSION=1.11.4 SIGNING_KEY=A1C052F8 VTS_VERSION=0.1.10 BUILD_PATH=/tmp/build PREFIX=/opt/router \
+	&& export NGINX_VERSION=1.11.4 OPENSSL_VERSION=1_0_2i SIGNING_KEY=A1C052F8 VTS_VERSION=0.1.10 BUILD_PATH=/tmp/build PREFIX=/opt/router \
 	&& rm -rf "$PREFIX" \
 	&& mkdir "$PREFIX" \
 	&& mkdir "$BUILD_PATH" \
 	&& cd "$BUILD_PATH" \
+	&& get_src 7e07311e7a05750e015f78b216c6ec9e4596a21a50eb93d3dc7a608b9900b3c0 \
+		"https://github.com/openssl/openssl/archive/OpenSSL_$OPENSSL_VERSION.tar.gz" \
+	&& cd "$BUILD_PATH/openssl-OpenSSL_$OPENSSL_VERSION" \
```

Comment:
You could use `pushd` instead of `cd` here, and `popd` instead of `cd ..` down below. Then this code doesn't assume as much about the directory nesting level.